Code/Tools/SceneAPI/SceneCore/Tests/Utilities/CoordinateSystemConverterTests.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzTest/AzTest.h>
+#include <AzCore/Math/Matrix3x4.h>
+#include <SceneAPI/SceneCore/Utilities/CoordinateSystemConverter.h>
+
+namespace EMotionFX
+{
+    const AZ::Vector3 sourceBasisVectors[3] = {
+        AZ::Vector3(1.0f, 0.0f, 0.0f),
```

Comment:
nitpick - for this kind of thing I think it would slightly better to use AZ::Vector3::CreateAxisX() etc. With a big list of float values it can be easy to make a typo, copy-paste error etc, and I find the CreateAxisX() easier to parse.